BraintreeCore/src/main/java/com/braintreepayments/api/ShopperInsightsClient.kt

```diff
@@ -0,0 +1,37 @@
+package com.braintreepayments.api
+
+import androidx.annotation.VisibleForTesting
+
+/**
+ * Use [ShopperInsightsClient] to optimize your checkout experience
+ * by prioritizing the customer’s preferred payment methods in your UI.
+ * By customizing each customer’s checkout experience,
+ * you can improve conversion, increase sales/repeat buys and boost user retention/loyalty.
+ * - Note: This feature is in beta. It's public API may change in future releases.
```

Comment:
This ` - Note: ` syntax is specific to Swift/Xcode to get the below formatting. Not sure if there's an equivalent in Android


![Screenshot 2023-12-06 at 11 33 58 AM](https://github.com/braintree/braintree_android/assets/35243507/8640b4fe-80de-46d5-b72e-565c86c11445)